modules/client/src/multisig/internal/utils.ts

```diff
@@ -26,6 +25,9 @@ export function toMultisigProposal(
   const endDate = new Date(
     parseInt(proposal.endDate) * 1000,
   );
+  const executionDate = new Date(
+    (parseInt(proposal.executionDate) || 0) * 1000,
```

Comment:
If for some reason it was `0` or empty, I would leave the whole Date as null. A proposal may not have been executed. Yet, creating a `new Date(0)` will produce a valid date in the 70's I think